### ISSUE TITLE
Making a mod to add sounds/taunt to the actual mod

### ISSUE THREAD

**Issue by SpLinTRTV** [gogozooom/Hide-and-Seek-LC-#6]
Hi again ! I'm coming to ask you if you are okay with me making (i'm trying but still didn't suceed to this day since I'm ass in code) a mod to add other sounds/taunt to your mod ?

I'm french, and most of the memes/taunt you use doesn't ring the same to me and my friends, so, I made a new bunch of taunt using french memes and sent it to my friends. It made our session even funnier, and since I like to make things automatic a possible I wanted to make a mod to allow moving my sound folder into yours (even tho I can't make it work outside of visual studio).

I think it might make people download your mod even more if they were able to make their own list of taunt in some ways, since internet culture is different based on the language and age.

I hope I'm not bothering you with that and hope we can find a way to make it work if you are okay with it ! Thanks !

**Comment by SpLinTRTV** [comment]
I managed to make a code that works fine. With two version, one that will simply copy the file to add taunt to yours and one that will replace them in case the player only want the custom ones.
And you can just take my mod, rename it in the manifest and dll the way you want to fit your playstyle or language ect... and simply add the sound you want to add or to replace. It could be neat to have custom sounds packs.

Of course, if you don't want it. Then it won't happen.

**Comment by gogozooom** [comment]
Adding built-in control over the taunts would be cool. I could rework how the sounds are loaded to make sound packs more convenient to share and load. And I can design the sound packs to include control over the credit rewards, in case someone wants to nerf or buff rewards for specific taunts.
I might also attempt transferring and loading a taunt pack from the host, and appropriate fail safe checks, to remove the requirement to share the file manually entirely.

I will remember to add this to my to-do list!

**Comment by SpLinTRTV** [comment]
yeah kinda like Clementinise- CustomSounds mod ? The mod go through every plugins folder to find the folders named "CustomSounds", it would simplify everything and if possible have an option to use multiple sound pack/use only the one you want.
